packages/ble-transport/src/transports/BleInboundTransport.ts

```diff
@@ -0,0 +1,55 @@
+import type { Central } from '@animo-id/react-native-ble-didcomm'
+import type { Agent, InboundTransport, Logger } from '@aries-framework/core'
+import type { EmitterSubscription } from 'react-native'
+
+import { utils } from '@aries-framework/core'
+import { MessageReceiver } from '@aries-framework/core/build/agent/MessageReceiver'
+import { isValidJweStructure, JsonEncoder } from '@aries-framework/core/build/utils'
+
+import { BleTransportSession } from './BleTransportSession'
+
+export class BleInboundTransport implements InboundTransport {
+  public supportedSchemes: string[] = ['ble']
+  private agent!: Agent
+  private central: Central
+  private logger!: Logger
+  private listener!: EmitterSubscription
+
+  public constructor(sdk: Central) {
+    this.central = sdk
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public async start(agent: Agent<any>): Promise<void> {
```

Comment:
```suggestion
  public async start(agent: Agent): Promise<void> {
```

I believe this should work.